magefile.go

```diff
@@ -1565,6 +1567,15 @@ func shouldBuildAgent() bool {
 	return ret
 }
 
+func timestampEnabled() bool {
+	timestamp := os.Getenv("TEST_INTEG_TIMESTAMP")
+	if timestamp == "" {
+		return false
```

Comment:
Should we have this enabled by default? There have been lots of discussion about how to make the test time shorter, make it more obvious where the time is being spent by default would help direct those suggestions to the right places.

Reply:
I prefer not to have it on by default, I don't like to look at them. I think overall it makes the output less clean and ugly. Would look for majority rule on switching to default enabled.